lib/resources/r4/allergy_intolerance.yaml

```diff
@@ -113,18 +99,32 @@ fields:
   cardinality: 0..*
   type: BackboneElement
   description: Adverse Reaction Events linked to exposure to substance
-  url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.code
+  url: https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.code
   children:
 
   - name: manifestation
     required: 'Yes'
     cardinality: 1..*
     type: CodeableConcept
     description: Clinical symptoms/signs associated with the Event
-    url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.reaction.manifestation
+    url: https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.reaction.manifestation
     binding:
       description: Clinical symptoms and/or signs that are observed or associated with the adverse reaction event.
       terminology:
       - display: SNOMED CT Clinical Findings
         system: http://snomed.info/sct
         info_link: http://hl7.org/fhir/r4/valueset-clinical-findings.html
+
+  - name: severity
+    required: 'No'
+    cardinality: 0..1
+    type: code
+    description: mild | moderate | severe (of event as a whole)
+    url: https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.reaction.severity
+    binding:
+      description: Clinical assessment of the severity of a reaction event as a whole, potentially considering multiple
```

Comment:
Can we only say 'Clinical assessment of the severity of a reaction event as a whole' and not mention the later. Since, we only support writing in one manifestation.

Thoughts @JFisk42

Reply:
@praneethk2094 @JFisk42 this thread does not have a resolution

Reply:
Update was made with commit: https://github.com/cerner/fhir.cerner.com/pull/321/files#diff-eeda3e608dc0db5d6d7c877900383353R122